tests/core/test_nlg.py

```diff
@@ -296,6 +296,68 @@ def test_nlg_fill_template_button(button_slot_name, button_slot_value):
     }
 
 
+@pytest.mark.parametrize(
```

Comment:
what do you think of condensing all three tests to one using `pytest.mark.parametrize`?